src/main/src/main/java/org/geoserver/systemstatus/BaseSystemInfoCollector.java

```diff
@@ -2,7 +2,7 @@
  * This code is licensed under the GPL 2.0 license, available at the root
  * application directory.
  */
-package org.geoserver.status.monitoring.collector;
+package org.geoserver.systemstatus;
```

Comment:
I read you proposition to quickly, IMHO it should be ``org.geoserver.system.status`` instead of ``org.geoserver.systemstatus``.

Reply:
Refactored.